Cargo.toml

```diff
@@ -2,6 +2,11 @@
 authors = ["49nord GmbH <marc@49nord.de>"]
```

Comment:
This needs to be `hello@49nord.de`. =)

Reply:
(my bad)

Reply:
Done.
(Very importantly, that also made the commit hash much nicer.)